sections/02_intro.md

```diff
@@ -23,31 +23,47 @@ algorithms have produced comparable or higher accuracy than previous
 best-in-class methods that required years of extensive customization, and
 specialized implementations are now being used at industrial scales.
 
+Deep learning approaches grew from research in neural networks.
 Neural networks were first proposed in 1943 [@doi:10.1007/BF02478259] as a model
 for how our brains process information. The history of neural networks is
 interesting in its own right [@doi:10.1103/RevModPhys.34.135]. In neural
 networks, inputs are fed into a hidden layer, which feeds into one or more
-hidden layers, which eventually produce an output layer. The neural networks
+hidden layers, which eventually produce an output layer. 
+A layer consists of a set of nodes, usually called "features" or "units,"
+which are connected via edges to the immediately earlier and the
+immediately deeper layers.
+The nodes of the input layer generally consist of the variables being measured in the dataset
+of interest - for example, each node could represent the intensity value of a specific
+pixel in an image processing application or a gene expression value in a transcriptomics 
+experiment.
+The neural networks
 used for deep learning have multiple hidden layers. Each layer essentially
 performs feature construction for the layers before it. The training process
 used often allows layers deeper in the network to contribute to the refinement
 of earlier layers. For this reason, these algorithms can automatically engineer
 features that are suitable for many tasks and customize those features for one
-or more specific tasks. Deep learning does many of the same things as more
-familiar approaches. Like a clustering algorithm, it can build features that
-describe recurrent patterns in data. Like a regression approach, deep learning
-methods can predict some output. However, deep learning methods combine both of
-these steps. When sufficient data are available, these methods construct
+or more specific tasks. 
+
+Deep learning does many of the same things as more
+familiar machine learning approaches. In particular, deep learning approaches can be used both in *supervised*
+applications - where the goal is to accurately predict one or more labels or outcomes associated with each
```

Comment:
Is there an emdash vs endash issue, i.e. -- vs -? Raised this earlier and not sure how it resolved.

Reply:
Just opened a new issue https://github.com/greenelab/deep-review/issues/695 to ask about that.

Reply:
Looks like it should the em dash cf https://github.com/greenelab/deep-review/issues/592 https://github.com/greenelab/deep-review/issues/695